modules/running-compliance-scans-worker-node.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/compliance-scans.adoc
+
+[id="running-compliance-scans-worker-node_{context}"]
+=  Scheduling the result server pod on a worker node
+
+The result server pod is used to mount the persistent volume (PV) storage volume that stores the raw Asset Reporting Format (ARF) scan results. The `nodeSelector` and `tolerations` attributes allow you to configure the location of the result server pod.
+
+This is helpful for those environments where control plane nodes are not permitted to mount persistent volumes.
+
+.Procedure
+
+. Inspect the `ScanSetting` object by running:
++
+[source,terminal]
+----
+$ oc describe scansettings default -n openshift-compliance
```

Comment:
Note the name of the ScanSetting below -- it's not default, just based on the default with the minimal changes needed to run on workers. I think explaining the default SS is a good idea unless we already do it elsewhere in the doc, but the current flow gives the idea that the example describes the `scansetting/default` object wihch is not the case.

Reply:
Resolved